### ISSUE TITLE
Output dynamodb and s3 backends

### ISSUE THREAD

**Issue by nitrocode** [aws-ia/terraform-aws-control_tower_account_factory#81]
It would be nice to retrieve the dynamodb tables and s3 backends

```
⨠ terraform state list | grep "aws_s3_bucket\..*backend"
module.control_tower_aft.module.aft_backend.aws_s3_bucket.primary-backend-bucket
module.control_tower_aft.module.aft_backend.aws_s3_bucket.secondary-backend-bucket
⨠ terraform state list | grep "dynamodb_table\.lock-table"
module.control_tower_aft.module.aft_backend.aws_dynamodb_table.lock-table
```

without having to do a `terraform state show` for each of the above resources

```
terraform state show module.control_tower_aft.module.aft_backend.aws_s3_bucket.primary-backend-bucket
terraform state show module.control_tower_aft.module.aft_backend.aws_s3_bucket.secondary-backend-bucket
terraform state show module.aft.module.aft_backend.aws_dynamodb_table.lock-table
```

**Comment by balltrev** [comment]
Thanks for the request @nitrocode. Just to confirm, you're looking at having these exposed as outputs correct? I've gone ahead and made a backlog in regards to that, but just wanted to make sure I got it right.

**Comment by nitrocode** [comment]
Yes, it would be nice to have these exposed as outputs from the module.

**Comment by JamesActually** [comment]
With AFT [1.15.0](https://github.com/aws-ia/terraform-aws-control_tower_account_factory/releases/tag/1.15.0), the DynamoDB lock table and S3 backend buckets are now exposed as outputs (`aft_backend_lock_table_name`, and `aft_primary_backend_bucket_id`/`aft_secondary_backend_bucket_id`, respectively).  This is in addition to other new outputs for DynamoDB table, IAM role, S3 bucket name, KMS Key, Step Function, and SNS Topic ARNs.

The outputs are listed in [outputs.tf](https://github.com/aws-ia/terraform-aws-control_tower_account_factory/blob/main/outputs.tf) as well as [README.md](https://github.com/aws-ia/terraform-aws-control_tower_account_factory/blob/main/README.md#outputs).

I recommend installing the latest release if you want to make use of these outputs.
